.markdown_link_check_config.json

```diff
@@ -6,8 +6,12 @@
   ],
   "replacementPatterns": [
     {
-        "pattern": "^/",
-        "replacement": "{{BASEURL}}/"
+      "pattern": "^/",
+      "replacement": "{{BASEURL}}/"
+    },
+    {
+      "pattern": "^https://github.com/open-telemetry/opentelemetry-proto/(blob|tree)/[^/]+/docs/",
+      "replacement": "LINK-CHECK-ERROR-USE-LOCAL-PATH-TO-DOC-PAGE-NOT-EXTERNAL-URL/"
```

Comment:
Can you clarify what does this replacement do?

Reply:
External URLs to local doc pages will be replaced by the prefix `LINK-CHECK-ERROR-USE-LOCAL-PATH-TO-DOC-PAGE-NOT-EXTERNAL-URL`, which of course, will fail the link-checker. Local pages should be referred to using a local path reference, not an external link.